engine/access/state_stream/backend.go

```diff
@@ -91,6 +95,7 @@ func New(
 		broadcaster:     broadcaster,
 		rootBlockHeight: rootHeight,
 		rootBlockID:     rootBlockID,
+		highestHeight:   atomic.NewUint64(0),
```

Comment:
better to use `counter.NewMonotonousCounter`

Reply:
do we persist this value? 

If not, after restart, it won't be able to return any execution data until it has downloaded execution data for a new block. 

This might also be a problem for historical EN, because there is no new blocks, which means the highestHeight will stay at 0?

Reply:
added logic to populate the highest height on startup in [d369668](https://github.com/onflow/flow-go/pull/4383/commits/d36966842db162ccc21237e4ae2547bc15dac4fe)